File: content/03/code/js/03-matching-an-integer-value/features/step_definitions/steps.js
const { Given, When, Then } = require('cucumber')

Given('Lucy is located {int} metres from Sean', function (distance) {
  // Write code here that turns the phrase above into concrete actions
  console.log(`Lucy is ${distance * 100} centimetres from Sean`);

  return 'pending'
})

When('Sean shouts "free bagels at Sean\'s"', function () {
  // Write code here that turns the phrase above into concrete actions
  return 'pending'
})

Then('Lucy hears Sean’s message', function () {
  // Write code here that turns the phrase above into concrete actions
  return 'pending'
})

File: content/bdd-with-cucumber/03/code/ruby/06-allow-for-singular-or-plural-metres/features/step_definitions/steps.rb
require 'shouty'

Given("Lucy is located {int} metre(s) from Sean") do |distance|
  # TODO: automation code to place Lucy and Sean goes here
  pending "Lucy is #{distance * 100} centimetres from Sean"
end

When("Sean shouts {string}") do |message|
  @sean.shout(message)
  @message_from_sean = message
end

Then("Lucy hears Sean's message") do
  expect(@lucy.messages_heard).to eq [@message_from_sean]
end
